internal/pkg/externalplugins/config.go

```diff
@@ -17,6 +17,8 @@ const (
 
 // Configuration is the top-level serialization target for external plugin Configuration.
 type Configuration struct {
+	TiChi string `json:"tichi-web,omitempty"`
```

Comment:
The name of this configuration does not feel like a good description of the content of the configuration item.

Reply:
TiChiWeb?

Reply:
Renamed.